.github/workflows/release-app.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish APP to Hub
+
+on:
+  push:
+    branches:
+      - main
+    tags: 
+      - "^v[0-9]+.[0-9]+.[0-9]+-app$"
+
+jobs:
+  hub:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Pre-Bootstrap
+        run: make pre-bootstrap
+
+      - name: Build APP production image
+        run: >
+          WARREN_APP_IMAGE_BUILD_TARGET=production \
+          WARREN_APP_IMAGE_TAG=$GITHUB_SHA \
+          WARREN_FRONTEND_IMAGE_BUILD_TARGET=production \
+            make build-docker-app
+            
+      - name: Check built image availability
+        run: docker images "warren-tdbp:*"
+
+      - name: Tag image
+        run: >
+          docker images apuiavignonuniversity/warren-tdbp
+          DOCKER_TAG=$([[ "$GITHUB_REF_TYPE" == "branch" ]] && echo $GITHUB_REF_NAME || echo $GITHUB_REF_NAME | sed 's/^v//')
+          # Display either:
+          # - DOCKER_TAG: main (Git branch main)
+          # or
+          # - DOCKER_TAG: 1.1.2 (Git tag v1.1.2)
+          echo "DOCKER_TAG: ${DOCKER_TAG} (Git $GITHUB_REF_TYPE $GITHUB_REF_NAME)"
+          docker tag warren-tdbp:$GITHUB_SHA apuiavignonuniversity/warren-tdbp:app-$DOCKER_TAG
+          # Tag as "latest" for stable release git tags
+          if [[ "$GITHUB_REF_TYPE" == "tag" ]]; then
+              docker tag warren-tdbp:$GITHUB_SHA apuiavignonuniversity/warren-tdbp:app-latest
+          fi
```

Comment:
```suggestion
          [[ "$GITHUB_REF_TYPE" == "tag" ]] && docker tag warren-tdbp:$GITHUB_SHA apuiavignonuniversity/warren-tdbp:app-latest
```

Reply:
I agree with the one-liner, but the line is quite long already :sweat_smile:
I think I'll leave it as is, if you don't mind!